src/aatoolbox/datasources/fewsnet/fewsnet.py

```diff
@@ -0,0 +1,233 @@
+"""
```

Comment:
Well, with every new folder you are actually creating a new package. It could be a bit of overkill to have only one module per package. However, if this module grows and splits into multiple files then it makes more sense. 

I think another disadvantage of overly hierarchical structure is then the imports get overly complex. But I [realized](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces) that we are actually suppose to define all public interfaces in `__all__`, so this could be a work around (I'm still trying to get my head around it a bit though)